client/src/components/application/Transactions.vue

```diff
@@ -148,149 +158,161 @@
 </template>
 
 <script>
-  import { mapState } from 'vuex'
-  import moment from 'moment'
-  import CompNameIcon from './Examine/CompNameIcon'
-  import Spinner from './spinner'
+import { mapState } from 'vuex';
```

Comment:
Why un-indent?

Why add trailing semi-colons when not needed?

Reply:
Both of these i think editor did based on the predetermined linting rules. Will go through and fix.